java-manta-client-unshaded/src/main/java/com/joyent/manta/client/MantaObjectMapper.java

```diff
@@ -47,9 +46,7 @@ public MantaObjectMapper() {
                 .without(DeserializationFeature.FAIL_ON_IGNORED_PROPERTIES);
         setConfig(deserializationConfig);
 
-        SerializationConfig serializationConfig = getSerializationConfig()
-                .with(SerializationFeature.WRITE_EMPTY_JSON_ARRAYS)
```

Comment:
Are these settings now default?

Reply:
We were never doing serialization where these properties were at play and the default value was that this was enabled.